### PR TITLE
Typo fix in target_gather docs

### PR DESCRIPTION
**Pull request by Anirban0011**
# What this PR does:

* Adds minor updates to the docstring of utils/target_gather.py

### CODE REVIEW

**Comment by Anirban0011** [comment]
@ianstenbit @LukeWood , Kindly check the above PRs if you have time, I have modified some of the definitions in some of the existing docstrings , would like to know what you think about it, please let me know if there's anything you want to know about.

File: keras_cv/utils/target_gather.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _target_gather(
        Tensor representing the masking for each target. `True` means the corresponding
        entity should be masked to `mask_val`, `False` means the corresponding entity
        should be the target value.
-     mask_val: optinal float representing the masking value if `mask` is True on
+     mask_val: optional float argument representing the masking value if `mask` is True on
```
Comment (ianstenbit) [inline keras_cv/utils/target_gather.py:39]: nit: no need for "argument" here -- it's implied.

**Review by ianstenbit — CHANGES_REQUESTED** [review]
Thanks for the fix!

**Comment by ianstenbit** [comment]
@Anirban0011 thanks for sending the docs cleanup PRs.

Could you please update PR names to describe what's changing?
(e.g. instead of "update some_component.py", say "Typo fix in SomeComponent docs")

File: keras_cv/utils/target_gather.py (hunk 1 above)
Comment (Anirban0011) [reply keras_cv/utils/target_gather.py:39]: ok, fixing it!

**Comment by Anirban0011** [comment]
> @Anirban0011 thanks for sending the docs cleanup PRs.
> 
> Could you please update PR names to describe what's changing? (e.g. instead of "update some_component.py", say "Typo fix in SomeComponent docs")

Ok , will do it immediately.

**Review by ianstenbit — APPROVED** [review]
Thanks!

**Comment by Anirban0011** [comment]
@ianstenbit  I have updated other PR names accordingly, have a look. Thanks!

**Comment by Anirban0011** [comment]
@ianstenbit /gcbrun kindly. Thanks and have a look at the above referenced doc fix PRs.

**Pull request merged**